PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -19,24 +21,58 @@ struct StoreAppView: View {
     @State var warningMessage: String?
 
     @EnvironmentObject var downloadVM: DownloadVM
+    @EnvironmentObject var installVM: InstallVM
 
     var body: some View {
-        StoreAppConditionalView(selectedBackgroundColor: $selectedBackgroundColor,
-                                selectedTextColor: $selectedTextColor,
-                                selected: $selected,
-                                app: app,
-                                isList: isList,
-                                warningSymbol: $warningSymbol,
-                                warningMessage: $warningMessage)
-        .gesture(TapGesture(count: 2).onEnded {
-            if let url = URL(string: app.link) {
-                downloadApp(url, app)
+        Group {
+            if #available(macOS 13.0, *) {
```

Comment:
Does the <13.0 solution work without caveats on 13.0? If so we should just stick with the one and avoid the `#available` all together

Reply:
I may rework DetailedView to make the library available in macOS 12 too so if I couldn't, I'll get back to if #available parts. But for explaining, macOS 12 has onTapGesture and simultaneousGesture that you can't put in any if statement so it is necessary to wrap it in `if #available`